components/automate-cli/cmd/chef-automate/cmdUtils.go

```diff
@@ -140,14 +142,19 @@ func (c *remoteCmdExecutor) executeCmdOnGivenNodes(input *CmdInputs, nodeIps []s
 	}
 	timeout := input.WaitTimeout
 	inputFileToOutputFileMap := map[string]string{}
-	for _, file := range inputFiles {
-		destinationFile := remoteService + "_" + timestamp + "_" + file
-		if strings.Contains(file, "/") {
-			filePath := strings.Split(file, "/")
-			lastFileidx := len(filePath) - 1
-			destinationFile = filePath[lastFileidx]
+
+	if input.SourceConfig != "" && input.DestinationConfig != "" {
```

Comment:
This approach is different, we can pass the prefix of destination files, and then use that in all locations.

Please connect with me, I can tell you more about this.